Kitodo/src/main/resources/kitodo_config.properties

```diff
@@ -163,6 +163,10 @@ ImagePrefix=\\d{8}
 #       image names, all characters are supported
 ImageSorting=number
 
+# Insert a path to an tif image which is taken as dummy images for a process
+# in order to start with pagination editing.
+dummyImagePath=C:/dev/local/images/dummyImage.tif
```

Comment:
Please don't use windows-style paths in kitodo config. You can use them in your local config.